src/jit/morph.cpp

```diff
@@ -2813,9 +2814,7 @@ GenTreeCall* Compiler::fgMorphArgs(GenTreeCall* callNode)
             assert(arg2 != nullptr);
             nonStandardArgs.Add(arg2, REG_LNGARG_HI);
         }
-#endif // !defined(LEGACY_BACKEND) && defined(_TARGET_X86_)
-
-#if !defined(LEGACY_BACKEND) && !defined(_TARGET_X86_)
+#else // defined(_TARGET_X86_)
```

Comment:
(Minor) Here the comment should read as

#else // !defined(_TARGET_X86_)

to indicate #else marks the beginning of the code where it is not meant for x86.